homeassistant/components/dynalite/__init__.py

```diff
@@ -117,7 +117,7 @@ def validate_area(config: Dict[str, Any]) -> Dict[str, Any]:
     vol.All(
         {
             vol.Required(CONF_NAME): cv.string,
-            vol.Optional(CONF_TEMPLATE): cv.string,
+            vol.Optional(CONF_TEMPLATE): vol.In(DEFAULT_TEMPLATES.keys()),
```

Comment:
`keys()` is the default iterator.
```suggestion
            vol.Optional(CONF_TEMPLATE): vol.In(DEFAULT_TEMPLATES),
```

Reply:
done